sqlmesh/dbt/loader.py

```diff
@@ -119,7 +122,11 @@ def _load_project(self) -> Project:
             return self._project
 
         self._project = Project.load(
-            DbtContext(project_root=self._context.path, target_name=self._context.gateway)
+            DbtContext(
+                project_root=self._context.path,
+                target_name=self._context.gateway,
+                schema_override=self._schema_override,
+            )
```

Comment:
Is schema_override being used anywhere except for project passing it into ManifestHelper? If not, why not pass it directly into `Project.load()`?

Reply:
Following the pattern of just passing the `context` everywhere. Can easily change this to what you're suggesting